lib/opal/nodes/literal.rb

```diff
@@ -241,6 +244,15 @@ def compile
 
         wrap '(', ')' if recv?
       end
+
+      def detect_and_remove_trailing_semicolon value, node
```

Comment:
Could you add parentheses around arguments? `def detect_and_remove_trailing_semicolon(value, node)`

Reply:
Sure, added.